### ISSUE TITLE
Packege Issue?

### ISSUE THREAD

**Issue by umlearn** [bunnieabc/Blender#11]
Can't find blender option after install.

**Comment by dkrape** [comment]
Hi umlearn,

Did you get a confirmation message like this when you unzipped and opened the plugin file?

<img width="445" src="https://user-images.githubusercontent.com/1051258/84639760-1fb0ed80-aef9-11ea-86d8-ba695198eda1.png">

Note that the current version has a bug which causes it to fail in some situations. You can find a workaround here: https://github.com/bunnieabc/Blender/issues/9

**Comment by umlearn** [comment]
Yes, I also checked Sketch plugin folder, Blender 2 file already there, But still can’t find blender in Sketch plugin menu. Weird...
Btw, My Sketch version is 66.

**Comment by dkrape** [comment]
Sorry, missed your reply (I'm not the owner of this package). Does it show in the list of Plugins when you click on "Manage plugins"? It is possible there was an installation problem.

**Comment by umlearn** [comment]
Can not find in "Manage plugin" either. 😢
